ext/authx/info.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<extension key="authx" type="module">
+  <file>authx</file>
+  <name>AuthX</name>
+  <description>Extended authentication module</description>
+  <license>AGPL-3.0</license>
+  <maintainer>
+    <author>Tim Otten</author>
+    <email>totten@civicrm.org</email>
+  </maintainer>
+  <urls>
+    <url desc="Main Extension Page">http://FIXME</url>
+    <url desc="Documentation">http://FIXME</url>
+    <url desc="Support">http://FIXME</url>
+    <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
+  </urls>
+  <releaseDate>2021-02-11</releaseDate>
+  <version>1.0</version>
+  <develStage>alpha</develStage>
+  <compatibility>
```

Comment:
@totten should we be hiding this extension from the GUI at this point or not?

Reply:
@seamuslee001 My vote would be to show it, but I don't feel super strongly in the first release with it.

Reply:
Yeh I guess the question I have is how supported is this but in any case I think its probably ok to show it